app/assets/stylesheets/application.scss

```diff
@@ -42,3 +42,22 @@
 .home-form {
   color: #ffffff;
 }
+
+.entries {
+  background-color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.